homeassistant/components/tado/__init__.py

```diff
@@ -35,40 +33,12 @@
 MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=10)
 SCAN_INTERVAL = timedelta(seconds=15)
 
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
Should we keep the schema around, but mark it deprecated?

```py
CONFIG_SCHEMA = cv.deprecated(DOMAIN)
```

If a user still has something in their configuration, they at least know why it isn't picked up.

Reply:
Good idea

`2021-01-21 15:42:34 WARNING (MainThread) [homeassistant.components.tado] The 'tado' option is deprecated, please remove it from your configuration`